.flake8

```diff
@@ -0,0 +1,2 @@
+[flake8]
+max-line-length = 120
```

Comment:
I'd like to stick to the PEP8 recommendation. (I think it's 79... but probably you can just remove this file?.)